ignite/pkg/goanalysis/goanalysis.go

```diff
@@ -211,3 +215,58 @@ func FormatImports(f *ast.File) map[string]string {
 	}
 	return m
 }
+
+// UpdateInitImports helper function to remove and add imports to an *ast.File.
+func UpdateInitImports(file *ast.File, importsToAdd, importsToRemove []string) ([]byte, error) {
```

Comment:
I'm not sure about this but I wanted to share that it might be worth to consider sending a writer instead of returning the altered file content.

Reply:
I will update the commend, init imports are imports only for we can init a package, using the underscore as a name, like the tools package is using: `_ "github.com/tendermint/....."`

Reply:
I'm returning a file content (`[]byte`) because `ast` removed all comments, and we need to add the build tag at the top of the file. If we rewrite the file direct from `ast.file` we will lose the build tag, and the chain will not compile

Reply:
> I will update the commend, init imports are imports only for we can init a package, using the underscore as a name, like the tools package is using: `_ "github.com/tendermint/....."`

Oh, true, that makes sense 👍